examples/README.md

```diff
@@ -15,3 +15,38 @@ $ kubectl create -f pod.yaml
 Other helper scripts:
 * `logs.sh` output of the plugin
 * `exec-bash.sh` logs into the plugin's container and runs bash
+
+## How to test RBD Snapshot feature
+
+Before continuing, make sure you enabled the required [feature gate](https://kubernetes-csi.github.io/docs/Setup.html#csi-volume-snapshot-support) in your Kubernetes cluster.
```

Comment:
I think the doc still points to kube1.12, as only one feature gate needs to be enabled, instead of the link we can add 
`--feature-gates=VolumeSnapshotDataSource=true` here

Reply:
Done